pkg/api/converterfromapi.go

```diff
@@ -549,6 +549,18 @@ func convertKubernetesConfigToVLabs(api *KubernetesConfig, vlabs *vlabs.Kubernet
 	vlabs.ClusterSubnet = api.ClusterSubnet
 	vlabs.NetworkPolicy = api.NetworkPolicy
 	vlabs.DockerBridgeSubnet = api.DockerBridgeSubnet
+	vlabs.NodeStatusUpdateFrequency = api.NodeStatusUpdateFrequency
```

Comment:
qq: for an existing containerService. it doesn't have any of these fields. Take `NodeStatusUpdateFrequency` as example. What will be used for `NodeStatusUpdateFrequency`? Default or Empty? And also what the behavior for Empty?